docs/updating-dependencies.md

```diff
@@ -30,10 +30,11 @@ semantic versioning via tags. Therefore the proper version is parsed from the ve
 branch and fixed using replace directive. Otherwise it will upgrade every time
 the command is started.
 
-When upgrading to a never version of OpenShift, this script have to be updated to
+When upgrading to a newer version of OpenShift, this script have to be updated to
 reflect the proper release.
 
 
+
```

Comment:
is this needed for something or is this just something that accidentally got in?

Reply:
Accidentally got added, removed it!